webcompat/db/__init__.py

```diff
@@ -4,15 +4,18 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+"""Databases initialization."""
+
+from hashlib import sha512
```

Comment:
Any reason this import is above `import os`?

Reply:
Ah, OK. I thought we grouped the "bare imports" (where you imported the entire module), then method imports.